jctools-core/src/main/java/org/jctools/util/PaddedAtomicLong.java

```diff
@@ -0,0 +1,355 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jctools.util;
+
+import java.util.function.LongBinaryOperator;
+import java.util.function.LongUnaryOperator;
+
+import static org.jctools.util.UnsafeAccess.UNSAFE;
+import static org.jctools.util.UnsafeAccess.fieldOffset;
+
+abstract class PaddedAtomicLongL1Pad extends Number implements java.io.Serializable {
+    private static final long serialVersionUID = 1;
+
+    transient long p01, p02, p03, p04, p05, p06, p07;
```

Comment:
please use byte padding as per the rest of JCTools

Reply:
Sure. Is there any differences compared to using longs?

Reply:
from JDK 15 the JVM started reordering fields to fill in gaps between object header and fields, resulting in `long` based padding fails.